src/Constants.php

```diff
@@ -107,6 +107,24 @@ class Constants extends \SimpleSAML\XML\Constants
     public const SIG_HMAC_SHA512 = 'http://www.w3.org/2001/04/xmldsig-more#hmac-sha512';
     public const SIG_HMAC_RIPEMD160 = 'http://www.w3.org/2001/04/xmldsig-more#hmac-ripemd160';
 
+    public static $RSA_DIGESTS = [
```

Comment:
Seems like you did create the static arrays, but you're not using them

Reply:
Actually, I did that on purpose, so that it would be absolutely explicit what we support per algorithm. But now that you have changed it it's fine too, not a big deal...